.circleci/config.yml

```diff
@@ -163,6 +163,19 @@ jobs:
                 --sessions "unit_tests(python_version='3.6')"
     working_directory: /var/code/googleapis/
 
+  php5:
```

Comment:
This may come later when we add more platforms, but we may want to call this `php55` or `php56` depending on which version of PHP is being used